File: engine/engine-model/src/main/scala/tools/packet/npctalk/GetDimensionalMirror.scala
package tools.packet.npctalk

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class GetDimensionalMirror(private var _talk: String) extends PacketInput {
  def talk: String = _talk

  override def opcode(): SendOpcode = SendOpcode.NPC_TALK
}

File: engine/engine-model/src/main/java/tools/packet/npctalk/GetDimensionalMirror.java
package tools.packet.npctalk;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record GetDimensionalMirror(String talk) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.NPC_TALK;
   }
}